src/App.css

```diff
@@ -238,25 +265,44 @@ Documentation styles below
   margin-top: 0;
 }
 
-.endpoints-list > li {
+.endpoints-list {
+  background-color:black;
+}
+
+.MuiExpansionPanelDetails-root > div > pre {
+  background: #2f3336
+}
+
+.endpoints-list > div {
   list-style-type: none;
   border-bottom: 1px solid #2f3336;
   padding: 12px 12px;
   margin: 0;
 }
 
-.endpoints-list > li > p {
+.endpoints-list > div > p {
   margin: 0;
   display: inline;
   margin-left: 16px;
 }
 
-.endpoints-list > li:hover {
+.endpoints-list > div:hover {
   background-color: #202224;
   border-left: 3px solid #effb31;
   padding-left: 10px;
 }
```

Comment:
I'm not seeing this hover state - might need to do some more selective class names. I'm happy to help troubleshoot!

Reply:
Yeah I'm not seeing what I am missing. It's hard with the material-ui components to capture which component I want to have the hover state. HELP PLZ.